hr_attendance_overtime/models/hr_employee.py

```diff
@@ -17,6 +18,9 @@ def todays_working_times(self, empl_domain):
         """Method used by my attendance/kiosk view in order
         to display employee planning and working times"""
         employee = self.search(empl_domain)
+        if not employee:
+            raise AccessError(_("Employee not found or not created for current user"))
```

Comment:
I think it's not matter of access control raising `odoo.exceptions.UserError` is probably better here.

Also could you add a unit test to cover this case as suggested by codecov ?

Reply:
thanks taking care of this